config/tests.conf.php

```diff
@@ -65,6 +69,9 @@
                     'type' => 'keyword',
                     'ignore_above' => 256,
                 ],
+                'referenced_resources' => [
```

Comment:
shall we call it `item_uris`? Cause a test can reference to items, workflow, groups, etc. But our use case is now only interested in `items`. It might bring confusion in the future and I do not see any use case where is useful to aggregate all the related resources in a single array. Wdyt?

Reply:
```suggestion
                'item_uris' => [
```